CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.0.1] - 2019-09-21
```

Comment:
Can you remove this heading and the blank line above it, keeping the "Unreleased" heading? We'll add in the appropriate heading at the time of release. Thanks!

Reply:
Ah, I see, now I get it :-). Yes, sure!

I also wrote "Fixed" instead of "Changed" now, since this is more of a fix instead of a change in functionality.